src/yang/ietf-bgp-common-structure.yang

```diff
@@ -129,7 +129,7 @@ submodule ietf-bgp-common-structure {
            for each neighbor.";
         reference
           "RFC 4456: BGP Route Reflection: An Alternative to
-                  Full Mesh.";
+                     Full Mesh.";
```

Comment:
Check indentation.

Reply:
This is *fixing* the broken indentation.

Reply:
Thought so, but it is hard to see a tab being replaced with spaces in a diff.